packages/@okta/vuepress-theme-prose/global-components/ApiAuthMethodWarning.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <blockquote>
+    <p><strong>Note</strong>: Some of the curl code examples on this page include SSWS API token authentication. However, Okta recommends using scoped OAuth 2.0 and OIDC access tokens to authenticate with Okta management APIs. OAuth 2.0 and OIDC access tokens provide fine-grain control over the bearer's actions on specific endpoints. See Okta API <a href="/docs/reference/core-okta-api/#authentication">authentication methods</a>.</p>
```

Comment:
Suggest putting "Okta API authentication methods" as the hyperlinked URL so the link is more comprehensively descriptive (good for a11y).